app/models/atheneum/event/user_changed.rb

```diff
@@ -0,0 +1,12 @@
+module Atheneum
+  module Event
+    class UserChanged
+      def self.execute!(payload)
+        body = payload.deep_symbolize_keys[:user]
+        body[:name] = "#{body.delete(:first_name)} #{body.delete(:last_name)}"
+        user = User.where('uid = ? or email = ?', body[:uid], body[:uid]).first_or_create(body.except(:permissions, :id))
+        user.set_permissions! body[:permissions]
```

Comment:
The or here has no sense. We should seaech by id. Rember that uid is not social id anymore.